algorithms/JavaScript/src/searching/binary-search-recursive.js

```diff
@@ -41,7 +41,7 @@ function binaryRecursive(arr, low, high, item) {
 }
 
 // returns 3 , Found at Index 3
-console.log(binaryRecursive([1, 3, 5, 7, 8, 9], 0, 5, 7));
+console.log(binaryRecursive([1, 3, 5, 7, 8, 9], 5, 7));
```

Comment:
It doesn't return 3, please check the method
<img width="782" alt="image" src="https://user-images.githubusercontent.com/37890026/145034523-28fb1636-026e-4633-8952-fdbd50ee79ec.png">

Reply:
Hi @ming-tsai, i have resolved the above mentioned issue, please check.